clang/tools/dpct/cmake/sycl_helper.cmake

```diff
@@ -0,0 +1,69 @@
+macro(GET_SOURCES _sources)
```

Comment:
//==---- dpct.hpp ---------------------------------*- C++ -*----------------==//
//
// Copyright (C) Intel Corporation
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
// See https://llvm.org/LICENSE.txt for license information.
//
//===----------------------------------------------------------------------===//